classes/options-general.php

```diff
@@ -456,21 +456,21 @@ class='howto'><?php echo $api_how_to; ?></span>
 
 													<tr valign="top">
 														<td><label
-																for="units"><?php _e( 'Distance Units', 'simplemap' ); ?></label>
+																for="units"><?php _e( 'Distance Units', 'SimpleMap' ); ?></label>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 129 characters

<!-- Comment by [SideCI](https://sideci.com) -->